src/index.jsx

```diff
@@ -23,8 +23,7 @@ class Tiers extends React.Component {
     let bundleId = this.state.bundleId;
     $.ajax({
       method: 'GET',
-      // url: '/tiersIncluded/:' + bundleId,
-      url: '/tiersIncluded',
+      url: '/tiersIncluded/' + bundleId,
       datatype: 'json',
       success: (data) => {
         this.setState({
```

Comment:
✓ Looks like you're doing 3 tiers and I agree with that for MVP! This is great though, grab the bundleId then query your database, setting the state of Tiers with the data. Then passing it down to, nice work